ansible_collections/arista/avd/roles/eos_cli_config_gen/README_v4.0.md

```diff
@@ -3215,21 +3215,24 @@ traffic_policies:
     counter_per_interface: < true | false >
   field_sets:
     ipv4:
-      < PREFIX FIELD SET NAME >:
-        - < IPv4 prefix 01>
-        - < IPv4 prefix 02>
-        - < IPv4 prefix 03>
+      - name: < PREFIX FIELD SET NAME >
+        prefix:
```

Comment:
since this is a list, we should use plural, `prefixes`. Same for ipv6.

Reply:
updated